### ISSUE TITLE
Lists of books lent to non-inventaire users

### ISSUE THREAD

**Issue by almereyda** [inventaire/inventaire#106]
If I'd want to lend a book to a non-inventaire user, I would need to name a shadow user and eventually invite it on the platform.

**Comment by maxlath** [comment]
yes! [related roadmap card](https://trello.com/c/Q1albgd0/92-also-track-books-lended-borrowed-to-people-without-account)
